examples/test_service/tests/test_deployment_failure.py

```diff
@@ -11,7 +11,8 @@
 SERVICE_NAME = "test-service"
 
 
-def test_full_cycle(project, remote_orchestrator):
+@pytest.mark.parametrize("fail", (True, False))
```

Comment:
I am failing to see the relation between the extension of this test case and your change, is there any?  Or is it simply an improvement over the existing test suite?

Reply:
Before this change the resource was set up to always fail and this is what the test verified. Adding the `import lorem` did not result in a test failure because the test expected the resource to fail. This change adds a case where the test expects the resource to succeed. I've run this test without the project install fix and it failed, so it is meant to verify that the project install works correctly.

Reply:
Right, I missed that it used to be checking it was failing, not succeeding

Reply:
I missed that for an embarrassingly long time myself, wondering why this test kept succeeding even when I added `raise Exception()` right at the top of the plugin file.